docs/HACKING.md

```diff
@@ -1,104 +1,181 @@
 WMCO  image from this base image:
+
 ```shell script
 make base-img
 make wmco-img IMG=$OPERATOR_IMAGE
 ```
+
 `make base-img` will not push the base image as a push is not needed. `make wmco-img` will make and push the
 `$OPERATOR_IMAGE`
 
-## Development workflow
-With the Operator image built and pushed to a remote repository, the operator can be deployed either directly,
-or through OLM.
+### Push an operator image
+
+Push your image to your container registry with podman
+
+`podman push quay.io/<quay username>/wmco:<operator name>`
+
+## Deploy
+
+Before deploying, ensure your environment is properly configured.
+
+#### Set private key
+
+in order to SSH into the instance, you'll need to set the `KUBE_SSH_KEY_PATH` to your private key.
 
-For each deployment method a kubeconfig for an OpenShift or OKD cluster must be exported
 ```shell script
-export KUBECONFIG=<path to kubeconfig>
+export KUBE_SSH_KEY_PATH=<path to pem>
 ```
 
-### Deploying the operator directly
+#### Set Image Name
+
+This is the name of the operator image that will be pushed to the container registry.
+
 ```shell script
-make deploy IMG=$OPERATOR_IMAGE
+export OPERATOR_IMAGE=quay.io/<quay username>/wmco:<identifying name>
 ```
-The above command will directly deploy all the resources required for the operator to run, as well as the WMCO pod.
-To remove the installed resources:
+
+#### Set CI to false
+
+This is to tell the tests that this test isn't running in CI.
+
 ```shell script
-make undeploy
+export OPENSHIFT_CI=false
 ```
 
-### Deploying the operator through OLM
+#### Set shared credentials file
+
+If you are running on AWS, set your shared credentials file.
+
 ```shell script
-hack/olm.sh run -k "<PRIVATE_KEY.PEM>"
+export AWS_SHARED_CREDENTIALS_FILE=<path to aws credentials file>
 ```
-Where `"<PRIVATE_KEY.PEM>"` is the private key which will be used to configure Windows nodes.
-This command builds the operator image, pushes it to remote repository and uses OLM to launch the operator. Executing
-the [Build](#build) step is not required.
 
-In order to build the operator ignoring the existing build image cache, run the above command with the `-i` option.
+#### Set KUBECONFIG
+
+To run tests, you should have a running cluster, and your KUBECONFIG set.
 
-To clean-up the installation, use:
 ```shell script
-hack/olm.sh cleanup
+export KUBECONFIG=<path to kubeconfig> 
 ```
 
-### Running e2e tests on a cluster
-The following environment variables must be set:
+The operator can now be deployed
+[Manually](#deploying-the-operator-manually),
+[Using the OLM](#running-the-olm-hack-script),
+[Using the e2e tests](#running-the-e2e-tests),
+[Using the e2e tests on platform agnostic infrastructure](#running-e2e-tests-on-platform-agnostic-infrastructure), or
+[mimicking the official deployment method](#deploying-mimicking-the-official-deployment-method).
+
+### Deploying the operator manually
```

Comment:
I think the header sizes for the diff deployment methods should all be the same size. The first three ([Manually](https://github.com/wgahnagl/windows-machine-config-operator/blob/WINC-959/docs/HACKING.md#deploying-the-operator-manually), [Using the OLM](https://github.com/wgahnagl/windows-machine-config-operator/blob/WINC-959/docs/HACKING.md#running-the-olm-hack-script), [Using the e2e tests](https://github.com/wgahnagl/windows-machine-config-operator/blob/WINC-959/docs/HACKING.md#running-the-e2e-tests)) are different from [Using the e2e tests on platform agnostic infrastructure](https://github.com/wgahnagl/windows-machine-config-operator/blob/WINC-959/docs/HACKING.md#running-e2e-tests-on-platform-agnostic-infrastructure), or [mimicking the official deployment method](https://github.com/wgahnagl/windows-machine-config-operator/blob/WINC-959/docs/HACKING.md#deploying-mimicking-the-official-deployment-method).